tox.ini

```diff
@@ -37,6 +37,7 @@ whitelist_externals = createdb
                       bash
 
 commands =
+         pip install --upgrade arcgis==1.6.2 --no-deps
```

Comment:
Could this rather be placed in setup, with rest of requirements?

Reply:
Hi @reinbach , i could not find a way to place this in the requirements file and also add the `--no-deps` flag. Unfortunately that flag is necessary, otherwise the lib gets installed with a lot of uneeded extras, and i don't see another way to enforce this flag. I did the same for the etools PR, i placed it in the Dockerfile..

I'll try to address the other suggestions, thanks for looking at it.